demisto_sdk/commands/format/update_playbook.py

```diff
@@ -53,11 +53,28 @@ def update_fromversion_by_user(self):
         """If no fromversion is specified, asks the user for it's value and updates the playbook."""
 
         if not self.data.get('fromversion', ''):
+
+            if self.assume_yes:
+                if self.verbose:
+                    if self.from_version:
+                        click.echo(f"Adding `fromversion: {self.from_version}`")
+
+                    else:
+                        click.echo(f"Adding `fromversion: {OLDEST_SUPPORTED_VERSION}`")
+                self.data['fromversion'] = self.from_version if self.from_version else OLDEST_SUPPORTED_VERSION
```

Comment:
does this work properly with stuff that should have specific fromversion (e.g. powershell should have 5.5) ? im pretty sure it was handled in the past

Reply:
I don't think it was considering how the code looked like before - there is a param called `from_version` which can be set by the user